### PR TITLE
Start FAQ section

### PR DESCRIPTION
**Pull request by tucksaun**

### CODE REVIEW

File: faq.rst [hunk 1]
```diff
@@ -0,0 +1,22 @@
+Frequently Asked Questions
+==========================
+
+.. note::
+
+   For questions related to biling, please head to the :doc:`billing` section.
```
Comment (fabpot) [inline faq.rst:6]: billing

File: faq.rst [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+What is accounted for storage quota?
```
Comment (fabpot) [inline faq.rst:8]: in the storage quota?

File: faq.rst [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+------------------------------------
+
+Your storage quota (as show by ``symfony project:info``) grants a maximum
```
Comment (fabpot) [inline faq.rst:11]: shown

File: faq.rst [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+of 5GB grants 5GB for each environment created.
+
+This quota is then shared by your :doc:`services </services/intro>` and
+:doc:`applications </config>`. Each one defines its own quota using the ``disk``
+configuration key. The sum of these quotas can not be bigger than your global
+project quota, in such case, an error message will be shown during ``symfony
+deploy``.
+
+The space used to store your source code, your application container images, and
+the temporary files (``/tmp``) does not account for the storage quota.
```
Comment (fabpot) [inline faq.rst:22]: so not

Comment (tucksaun) [reply faq.rst]: I don't understand your review 🤔

Comment (fabpot) [reply faq.rst]: do not account

Comment (tucksaun) [reply faq.rst]: Hum, "The space used [...] does not account". no?

Comment (fabpot) [reply faq.rst]: nvm, you're right

File: faq.rst [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+of 5GB grants 5GB for each environment created.
+
+This quota is shared by your :doc:`services </services/intro>` and
+:doc:`applications </config>`. Each one defines its own quota using the ``disk``
+configuration key. The sum of these quotas can not be bigger than your global
+project quota, in such case, an error message will be shown during ``symfony
+deploy``.
+
+The space used to store your source code, your application container images, and
+the temporary files (``/tmp``) does not account in the storage quota.
```
Comment (OskarStark) [inline faq.rst:22]: And is limited to ??MB

I ran into this issue because I thought it’s included

Comment (tucksaun) [reply faq.rst:22]: 4GB but I don't think it's well located in this question.
But I can add a new entry

**Pull request merged**